san/sanbase_graphql.py

```diff
@@ -94,18 +94,6 @@
     'mining_pools_distribution':{
         'query': 'miningPoolsDistribution',
         'return_fields': ['datetime', 'other', 'top10', 'top3']
-    },
```

Comment:
Why these are removed?

Reply:
Because when I've started making those functions for the queries, I thought, that every new function should be there, but it turned out, that it wasn't needed. Moreover, the test for the batch for equal format queries fail, because those 3 functions aren't equal format